### PR TITLE
Fix error on db-schema mismatch

### PR DESCRIPTION
**Pull request by pivnicek**
fixes: #86

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/MySQL.php [hunk 1]
```diff
@@ -139,8 +139,7 @@ public function export(array $table): array
         // if database set make sure the database and selected table schema match
         if (isset($table['table']) && $this->database && $this->database !== $table['table']['schema']) {
             throw new UserException(sprintf(
-                'Invalid Configuration in "%s".  The table schema "%s" is different from the connection database "%s"',
-                $table['name'],
+                'Invalid Configuration.  The table schema "%s" is different from the connection database "%s"',
```
Comment (Halama) [inline src/Keboola/DbExtractor/Extractor/MySQL.php]: The table name is no longer required in message because there is only one table in config row?

Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: yes, config rows don't have names under parameters, that's the root of the bug.

Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: I just got rid of it, but I guess better would be to change message on `isset($table['name']) ?:` maybe.

Comment (Halama) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: thanks, now I understand it better. So `$table['name']` is obsolete. What about to use `$table['tableName']`?

Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: good idea!

**Comment by pivnicek** [comment]
ok @Halama , I think this is good to go now.

**Comment by pivnicek** [comment]
@Halama ping:)

**Pull request merged**